tsdat/io/converters/string_to_datetime.py

```diff
@@ -64,6 +66,8 @@ def convert(
             self,
             data: xr.DataArray,
             variable_name: str,
+            # TODO: If this is in quotes because of a circular reference, that should be hashed out,
+            #  and this should be here just as a placeholder until that's resolved.
             dataset_config: "DatasetConfig",
```

Comment:
Yeah, I think circular imports were causing us grief, at least with the other repo layout. I wasn't really sure how to handle it, hence the postponed type annotation in quotes. How would you recommend resolving circular references?

Reply:
I tested just directly importing the class, and it seemed to work fine. See cbdff59c354c70470c55c0bb5c27e8238de652cc.